### DISCUSSION TITLE
Show name of provider, not just type

### DISCUSSION THREAD

**Discussion by mvdwetering** [music-assistant/support#2317] [Feature requests and ideas]
In some screens it shows "provider details" like in this screenshot it shows the "Filesystem (remote share)" provider. However I have multiple of these remote share providers (which I have given different names). It would be nice to see the name so I know which one it actually is. 

![image](https://github.com/music-assistant/support/assets/732514/42e26f99-1c69-46c3-ad41-dce940f914b8)

**Comment by OzGav** [comment]
Have you changed the name in the settings.?

**Reply by mvdwetering** [reply]
Yes 
![image](https://github.com/music-assistant/support/assets/732514/57672f94-e715-4b06-8015-2c4d6f5d07ae)

**Comment by OzGav** [comment]
This has been implemented in MA 2.4 if you supply a custom name for the provider.

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
